packages/malloy/src/api/stateless.spec.ts

```diff
@@ -315,6 +316,11 @@ describe('api', () => {
       const expected: Malloy.CompileQueryResponse = {
         result: {
           connection_name: 'connection',
+          annotations: [
+            {value: '#(test) hello\n'},
+            // TODO generated tags have no newline?
```

Comment:
Address TODO

Reply:
Annotations in the language contain the newline. So far, `new Tag().toString()` does not include a newline. It probably should to match?